.github/pull_request_template.md

```diff
@@ -8,8 +8,13 @@ Describe the changes you're proposing.
 
 ## Technical writer support
 
-Do you need a tech writer's support, for example to review your PR?  
+Do you need a tech writer's support, for example to review your PR?
 
 ## How to review
 
 Tell reviewers how to assess your changes.
+
+## Confirm
+
+- [ ] I have checked if any docs change here also requires updates to other repositories (ADRs / RFCs, README.md, Team Manual, Elsewhere in these docs)
```

Comment:
this will sound very nitpicky but I don't think Elsewhere needs a capital E could just be elsewhere I think